src/pages/rss.xml.ts

```diff
@@ -0,0 +1,100 @@
+// A fast, cached build-time only RSS.
```

Comment:
This one is a nit-picky, but what do you think about changing the name of this one to `spotlight.xml`? The `rss.xml` is a very generic one, and may lock us in the future. Maybe at some point we can have a blog, or notes, or events or some other entities that also expose a feed. By being more specific we can avoid some conflicts in the future.

Also, we need to link to this file in the html `head` section.

```
<link rel="alternate" type="application/rss+xml" title="NN1 Dev Club" href="https://nn1.dev/spotlight.xml">
```